tests/testdata/default_config.yaml

```diff
@@ -81,8 +81,10 @@ train:
   nproc_per_node: 1
   num_epochs: 10
   phased_mt_bench_judge: /cache/instructlab/models/prometheus-eval/prometheus-8x7b-v2.0
+  phased_phase1_effective_batch_size: 128
   phased_phase1_num_epochs: 10
   phased_phase1_samples_per_save: 25000
```

Comment:
Note: if you plan on adding the per-epoch saves as default here, might want to make the `phased_phase1_samples_per_save` and `phased_phase2_samples_per_save` a much higher number for now to avoid over-saving.

Reply:
meant for `src/instructlab/configuration.py` (can be done in #2004 alongside epoch addition)

Reply:
gotcha will do